src/lda.py

```diff
@@ -37,10 +37,10 @@ def print_topics(self, num=5):
 
         alphabet = self.corpus.alphabet
 
-        for t in xrange(self.T):
+        for t in range(self.T):
 
-            sorted_types = map(alphabet.lookup, argsort(nwt[:, t] + beta))
-            print 'Topic %s: %s' % (t+1, ' '.join(sorted_types[-num:][::-1]))
+            sorted_types = list(map(alphabet.lookup, argsort(nwt[:, t] + beta)))
+            print('Topic %s: %s' % (t+1, ' '.join(sorted_types[-num:][::-1])))
 
     def save_state(self, filename):
```

Comment:
This is so weird, I also have these changes locally. I think something must have happened because I confirmed these changes were present a while back. Regardless, I agree with your print statement changes and file reading changes.

Reply:
Sorry, the change I'm requesting is in the lines below. GitHub won't let me reference it. (so, not 42/43, but 49)